src/Augurk/Extensions/RavenDbServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Augurk.Api;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Indexes;
+using Raven.Embedded;
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class RavenDbServiceCollectionExtensions
+    {
+        public static void AddRavenDb(this IServiceCollection services)
+        {
+            var serverOptions = new ServerOptions
```

Comment:
Might want to check if this is still correct.

Reply:
What happened to our retention feature? Has that one been scrapped from Raven 4 or do we need to enable it in a different way?

Reply:
That's a good question, I haven't checked this yet. There seems to be some documentation on it [here](https://ravendb.net/docs/article-page/4.1/csharp/server/extensions/expiration). I'll add this to the checklist for the 3.0 release to test if this is still working so that we don't have to block this PR to wait for that.